src/make_ntuples.c

```diff
@@ -62,22 +62,12 @@ double get_tof(REC_Scintillator rsci, REC_Calorimeter  rcal, int pindex) {
     return tof;
 }
 
-int run(char * in_filename, bool use_simul,bool debug, int nevn, int run_no, double beam_E) {
-    double sf_params[NSECTORS][SF_NPARAMS][2] = {{{0.246,0.0140},{0.985,1.},{0.083,0.},{-0.031,0.}},
-                                                 {{0.255,0.0150},{0.985,1.},{0.043,0.},{-0.024,0.}},
-                                                 {{0.245,0.0130},{0.985,1.},{0.067,0.},{-0.025,0.}},
-                                                 {{0.245,0.0140},{0.985,1.},{0.048,0.},{-0.016,0.}},
-                                                 {{0.252,0.0130},{0.985,1.},{0.007,0.},{-0.007,0.}},
-                                                 {{0.242,0.0140},{0.985,1.},{0.095,0.},{-0.038,0.}}};
-    char*  out_filename = (char *) malloc(128 * sizeof(char));
-    if(!use_simul){
-        // Extract sampling fraction parameters.
-        if (get_sf_params(Form("../data/sf_params_%06d.root", run_no), sf_params)) return 8;
-        sprintf(out_filename, "../root_io/banks_%06d.root", run_no);
-    } else{
-        sprintf(out_filename, "ntuple_%s", in_filename);
-    }
+int run(char * in_filename, bool debug, int nevn, int run_no, double beam_E) {
+    double sf_params[NSECTORS][SF_NPARAMS][2];
+    if (get_sf_params(Form("../data/sf_params_%06d.txt", run_no), sf_params)) return 8;
 
+    char*  out_filename = (char *) malloc(128 * sizeof(char));
+    sprintf(out_filename, "../root_io/ntuples_%06d.root", run_no);
```

Comment:
This is wrong, but you had no way of knowing. The `ntuples.root` file *intentionally* does not indicate run number, since in the future it will be able to handle various input files with different run numbers. Please change it to how it is in the, master branch: `ntuples.root`.

It would be an useful feature in the future to allow the user to indicate an output filename (and indicate an input filename in `draw_plots`), but its not a priority for now, and I'd rather finish my thesis than worry about this small stuff.

Reply:
Reverted to original!